.github/workflows/release-docker.yml

```
@@ -70,8 +70,8 @@ jobs:
       - name: Get docker build arg
         id: build_arg
         run: |
-          echo "::set-output name=build_arg_name::"$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g')
-          echo "::set-output name=build_arg_value::"$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g')
+          echo "build_arg_name=$(echo ${{ github.event.inputs.package }}" | sed 's/\-/_/g') >> $GITHUB_OUTPUT
+          echo "build_arg_value=$(echo ${{ github.event.inputs.package }}" | sed 's/postgres/core/g') >> $GITHUB_OUTPUT
```

This quotation mark needs to move to right after `=` so that the GHA variable gets templated out within the echo expression before the sed command takes effect.

```suggestion
          echo "build_arg_name="$(echo ${{ github.event.inputs.package }} | sed 's/\-/_/g') >> $GITHUB_OUTPUT
          echo "build_arg_value="$(echo ${{ github.event.inputs.package }} | sed 's/postgres/core/g') >> $GITHUB_OUTPUT
```

---

Fixed. Thanks!